.github/workflows/cleanup.sh

```diff
@@ -20,4 +21,6 @@ kubectl delete customresourcedefinitions.apiextensions.k8s.io -l app.kubernetes.
 
 source ${FATS_DIR}/macros/helm-reset.sh
 
+NAMESPACE=${NAMESPACE-fats}
```

Comment:
rm

this script should never be invoked by anything other than the CI workflow.

Reply:
Why not? It's useful for testing. Seems harmless.